VERSION

```diff
@@ -1 +1 @@
-1.12.1
+1.13.0
```

Comment:
Should this be `1.13.1` since `1.13.0` was already released?

Reply:
Yeah, this is happening because `dev` is one commit behind `master`, and my code is off of `master`. So this PR contains the 1.13.0 version bump. I need to put up a separate pull request to catch `dev` up with `master`.

Once this fix is approved, we'll release it as `1.13.1`.

Reply:
See: #746